packages/react-components/react-virtualizer/src/hooks/useScrollRef.ts

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { debounce } from '../utilities/debounce';
+import { canUseDOM } from '@fluentui/react-utilities';
+import { ResizeCallbackWithRef } from './hooks.types';
+
+/**
+ * useScrollRef simplifies resize observer connection and ensures debounce/cleanup
+ */
+export const useScrollRef = (resizeCallback: ResizeCallbackWithRef) => {
```

Comment:
Do we think we should change this name to be more intuitive? It seems more about enabling easier hookup of RO and doesn't necessarily even have to be about scrolling parents?